Source/WebKit/Shared/mac/MediaFormatReader/MediaFormatReader.cpp

```diff
@@ -296,7 +297,7 @@ const void* MediaFormatReader::nextTrackReaderLogIdentifier(uint64_t trackID) co
 
 const void* MediaFormatReader::nextLogIdentifier()
 {
-    static uint64_t logIdentifier = cryptographicallyRandomNumber();
+    static uint64_t logIdentifier = cryptographicallyRandomNumber<uint32_t>();
```

Comment:
Left this as is, generating a 32-bit random number and storing it in a 64-bit global. New function name makes that a lot more obvious. Not sure if it’s a mistake.